tls/s2n_server_new_session_ticket.c

```diff
@@ -76,6 +87,31 @@ int s2n_server_nst_send(struct s2n_connection *conn)
     return 0;
 }
 
+S2N_RESULT s2n_tls13_server_nst_send(struct s2n_connection *conn, s2n_blocked_status *blocked)
+{
+    ENSURE_REF(conn);
+
+    if (conn->mode != S2N_SERVER || conn->actual_protocol_version < S2N_TLS13) {
+        return S2N_RESULT_OK;
+    }
+
+    ENSURE(conn->tickets_sent <= conn->tickets_to_send, S2N_ERR_INTEGER_OVERFLOW);
+    while (conn->tickets_to_send - conn->tickets_sent > 0) {
+        uint8_t nst_data[TLS13_MAX_NEW_SESSION_TICKET_SIZE] = { 0 };
+        struct s2n_blob nst_blob = { 0 };
+        struct s2n_stuffer nst_stuffer = { 0 };
+        GUARD_AS_RESULT(s2n_blob_init(&nst_blob, nst_data, sizeof(nst_data)));
+        GUARD_AS_RESULT(s2n_stuffer_init(&nst_stuffer, &nst_blob));
+
+        GUARD_AS_RESULT(s2n_tls13_server_nst_write(conn, &nst_stuffer));
+        nst_blob.size = s2n_stuffer_data_available(&nst_stuffer);
```

Comment:
We create nst_blob of size TLS13_MAX_NEW_SESSION_TICKET_SIZE, and initialize a stuffer for it on line 104. There's no way for the stuffer to write more than TLS13_MAX_NEW_SESSION_TICKET_SIZE, as it's not a growable stuffer. So essentially, because the stuffer can't be larger than TLS13_MAX_NEW_SESSION_TICKET_SIZE, neither can the ending blob size.